app/controllers/albums.js

```diff
@@ -44,3 +44,27 @@ exports.buy = (request, response) => {
     })
     .catch(error => handleError(error, 'Unexpected error when fetching'));
 };
+
+exports.boughts = (request, response) => {
+  if (!request.userLogged)
+    return response
+      .status(400)
+      .send(errors.authenticationError(['You must be logged to see the purchased albums']));
+  if (!request.userLogged.isAdmin && Number(request.params.user_id) !== request.userLogged.id)
```

Comment:
Same comment as in the previous PR, i think this kind of validations should be done in the middleware